### PR TITLE
Add to scater/scran Dockerfile

### LINKED ISSUE #17: Add packages to Dockerfile that have not been added yet
For the training modules, we need to add: 
- colorblindr 

Put this to a specific commit: 
```
devtools::install_github("clauswilke/colorblindr")
```

### PR DESCRIPTION
**Pull request by cansav09**
Addressing #18 and #17 , but probably don't close #17 just in case something else comes up later

Adding to the Dockerfile:
- `scran` and `scater` 
- `colorblindr`
- Update version of `alevinQC` with the directory fix

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -26,15 +26,20 @@ RUN apt-get update -qq && apt-get -y --no-install-recommends install \
   rjson \
   ggpubr \
   Seurat \
+  colorspace \
   && R -e "source('https://bioconductor.org/biocLite.R')" \
-  && R -e "BiocInstaller::biocLite(c('ensembldb', 'DESeq2', 'qvalue', 'org.Hs.eg.db', 'org.Dr.eg.db', 'ComplexHeatmap', 'ConsensusClusterPlus'), suppressUpdates = TRUE)"
+  && R -e "BiocInstaller::biocLite(c('ensembldb', 'DESeq2', 'qvalue', 'org.Hs.eg.db', 'org.Dr.eg.db', 'ComplexHeatmap', 'ConsensusClusterPlus'), suppressUpdates = TRUE)" \
+  && R -e "BiocInstaller::biocLite(c('scran', 'scater'))"
```
Comment (jaclyn-taroni) [inline Dockerfile]: Why aren't you suppressing updates in this one? I noticed on your sandbox PR as well.

Comment (cansav09) [reply Dockerfile]: `scater` and `scran` want an updated version of `rlang` and if you suppress updates and don't let them update `rlang` they don't work. There is likely a more sophisticated way to fix this, but this worked for me.

Comment (jaclyn-taroni) [reply Dockerfile]: I would recommend using [`devtools::install_version`](https://www.rdocumentation.org/packages/devtools/versions/1.13.6/topics/install_version) prior to this step and pinning the version of `rlang` that they want.

Comment (cansav09) [reply Dockerfile]: Pushed this change, but I am building the docker image now to make sure everything still is working fine.

Comment (jaclyn-taroni) [reply Dockerfile]: Why are you installing from Github rather than CRAN?

Comment (cansav09) [reply Dockerfile]: For rlang? IDK. I could do CRAN. Is that preferred?

Comment (jaclyn-taroni) [reply Dockerfile]: I don't see why we would want to use the development version of `rlang` and my recommendation above was to use `devtools::install_version` which installs a specified version of a CRAN package.

Comment (cansav09) [reply Dockerfile]: Noted. Will change and push. The docker image appears to have built successfully otherwise.

**Pull request merged**